tap-lightcast/tap_lightcast/tap.py

```diff
@@ -0,0 +1,46 @@
+"""lightcast tap class."""
+
+from typing import List
+from singer_sdk import Tap, Stream
+from singer_sdk import typing as th
+
+from tap_lightcast.streams import (
+    SkillsLatestVersion,
+    SkillsList,
+    SkillsDetails,
+)
+
+PLUGIN_NAME = "tap-lightcast"
+
+STREAM_TYPES = [
+    SkillsLatestVersion,
+    SkillsList,
+    SkillsDetails,
+]
+
+
+class TapLightcast(Tap):
+    """lightcast tap class."""
+
+    name = "tap-lightcast"
+    config_jsonschema = th.PropertiesList(
+        th.Property("client_id", th.StringType, required=True, description="Client ID"),
+        th.Property(
+            "client_secret", th.StringType, required=True, description="Client Secret"
+        ),
+        th.Property(
+            "limit",
+            th.IntegerType,
+            required=True,
+            description="Used for debugging purposes. It limits the number of IDs to select. Set this parameter to -1 to get all available IDs.",
```

Comment:
How is this going to work in prod? I set mine to -1 and its been running now for an hour